mlflow/models/evaluation/base.py

```diff
@@ -50,6 +50,20 @@
 
 _logger = logging.getLogger(__name__)
 
+MLFLOW_EVALUATE_LANGCHAIN_AUTOLOG_CONFIG = {
+    "log_input_examples": False,
+    "log_model_signatures": False,
+    "log_models": False,
+    "log_datasets": False,
+    "log_inputs_outputs": False,
+    "disable": False,
+    "exclusive": False,
+    "disable_for_unsupported_versions": True,
+    "silent": False,
+    "registered_model_name": None,
+    "extra_tags": None,
+}
```

Comment:
This seems to be unused. Can we move it to https://github.com/mlflow/mlflow/pull/11911/files#r1602160113?